test/distributed/test_c10d_logger.py

```diff
@@ -118,16 +118,14 @@ def test_exception_logger(self) -> None:
                 re.search("({.+})", captured.output[0]).group(0).replace("'", '"')
             )
 
-            self.assertEqual(len(error_msg_dict), 10)
+            self.assertEqual(len(error_msg_dict), 9)
```

Comment:
nit: remove this check entirely to reduce our maintenance overhead?

Reply:
why is that? This is part of a test. Having a proper test is not a maintenance overhead.